lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2095,6 +2095,50 @@ void SwiftLanguageRuntime::DidFinishExecutingUserExpression(
 
 bool SwiftLanguageRuntime::IsABIStable() { FORWARD(IsABIStable); }
 
+struct AsyncUnwindRegisters {
+  uint32_t async_ctx_regnum;
+  uint32_t fp_regnum;
+  uint32_t pc_regnum;
+  uint32_t dummy_regnum;
+};
+
+static llvm::Optional<AsyncUnwindRegisters>
+GetAsyncUnwindRegisters(llvm::Triple::ArchType triple) {
```

Comment:
It might be clearer if it says GetAsyncUnwindRegisterNumbers.  But I do love overly long variable & function names.

Reply:
that's better, I changed it.